koku/subs/test/test_subs_data_extractor.py

```diff
@@ -93,7 +93,7 @@ def test_determine_where_clause_and_params(self):
             "month": month,
         }
         expected_clause = (
-            "WHERE source={{source_uuid}} AND year={{year}} AND month={{month}} AND"
+            "WHERE source={{source_uuid}} AND year={{year}} AND month={{month}} AND "
```

Comment:
```suggestion
            "WHERE source={{source_uuid}} AND year={{year}} AND month={{month}} AND"
```
Maybe I'm miss-reading this :coffee: but does this line need the white space since the next line already has one? :slightly_smiling_face: " lineitem_productcode = double space probably doesn't hurt anyway, just saying.

Reply:
It does not need it, I removed the extra one.